src/utils.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+const RE_UUID = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
```

Comment:
This one is a bit more permissive than the server one (https://github.com/mozilla-services/cliquet/blob/master/cliquet/storage/generators.py#L54-L55)